src/widgets/courses/ui/courses.tsx

```diff
@@ -29,14 +43,18 @@ export const Courses = () => {
   const sortedCourses: Course[] = getActualData(sortParams);
 
   return (
-    <section className="rs-courses container" id="upcoming-courses">
-      <div className="rs-courses content">
-        <WidgetTitle>All courses</WidgetTitle>
-        <div className="rs-courses-wrapper" data-testid="courses-fancy">
+    <section className={cx('courses')} id="upcoming-courses" data-testid="courses">
```

Comment:
I investigated a little bit, and it seems to be that ```id="upcoming-courses"``` just copiedpasta form ```courses-school/ui/courses.txs```, and actually id here is not used. I would consider deleting it.

Reply:
I think you're right. I deleted it